### ISSUE TITLE
Xowa not changing date?

### ISSUE THREAD

**Issue by mortimer123** [gnosygnu/xowa#195]
Hi,

Great tool! Thanks.

My issue is that when I start the app it always uses July 1 for the date on the main page. By this I mean that the content for "On this day ..." isn't changing. The date on the system is changing.

Startup page is set to custom, "en.wikipedia.org/wiki/Main_Page".

Xowa 4.5.15.1709 from xowa.org.

Windows 7 64 bit.

Any ideas?

**Comment by gnosygnu** [comment]
Hi! Thanks for creating the ticket.

There are a few factors here

(1) The wikis from Download Central are static, which means the wiki data never changes. So, the Main Page will always be fixed at a certain date, which is the date I choose when I build the wiki.

(2) Unfortunately, this was a bug with the 2017-12 English Wikipedia build, where I forgot to update the date and it was stuck as 2017-07-01. It should have been 2017-12-01. I'll have this fixed for future builds, but for now, there's not much to do about it. I don't think it affects much outside the Main Page, but do let me know

This will be fixed for future builds, but for now, 2017-12 English Wikipedia will be stuck at 2017-07. This only affects a few pages like the Main Page. I don't think it has many other side-effects, but let me know if anything else is strange.

(3) If you really want to have a Main Page that reflects the current date, you can turn on Page Sync at home/wiki/Special:XowaCfg?grp=xowa.bldr.page_sync . This will download the latest static copy from the online Wikipedia. It's not a full offline situation, but it may be what you're looking for.

Hope the above makes sense.

Thanks.

**Comment by mortimer123** [comment]
Thank you for the quick reply!

This, the 12/17 download, is my second update to the data. The first, about a year ago was to download the XML dumps and create the data. A monumental piece of work. The date *did* update every day.

Page sync isn't an option. The system that has Xowa on in doesn't have internet access.

**Comment by gnosygnu** [comment]
Yeah, I tried to explain in https://github.com/gnosygnu/xowa/issues/196, but let me try again.

There are two wiki types: static (HTML) and dynamic (Wikitext)

A year ago, you downloaded the XML dumps. These allow you to build dynamic (Wikitext) pages which will always update the day (since they work just like online Wikipedia)

Now, you downloaded the 2017-12 Download Central databases. These are static (HTML) pages which will only show the day when the wiki was built (which I mistakenly set as 2017-07-01)

If you really want to have an up-to-date day, then I'd use the XML dumps. They actually are "better" than the HTML dumps since they work just like online Wikipedia works. However, they are slower and won't work on Android (because they are slower)

Hope this makes sense.
